numpy/core/code_generators/ufunc_docstrings.py

```diff
@@ -2875,19 +2880,20 @@ def add_newdoc(place, name, doc):
         Divisor array.
     out : tuple of ndarray, optional
         Arrays into which the output is placed. Their types are preserved and
-        must be of the right shape to hold the output. See doc.ufuncs.
+        must be of the right shape to hold the output.
 
     Returns
     -------
     out1 : ndarray
-        Element-wise result of floor division.
+        Element-wise quotient resulting from floor division.
     out2 : ndarray
         Element-wise remainder from division.
```

Comment:
If you're mentioning floor division above, then presumably you should do so here too?